lib/events.js

```diff
@@ -356,8 +379,11 @@ EventEmitter.prototype.removeListener =
     function removeListener(type, listener) {
       var list, events, position, i, originalListener;
 
-      if (typeof listener !== 'function')
-        throw new TypeError('"listener" argument must be a function');
+      if (typeof listener !== 'function') {
+        const errors = lazyErrors();
+        throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'listener',
+                                   'Function');
```

Comment:
Our docs mainly use `Function` but it is going to print `type Function` and this is somewhat inconsistent out of my perspective. I think we should stick to the lower cased version.